src/style.css

```diff
@@ -0,0 +1,92 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
```

Comment:
Fix linter mistakes, do not disable them. Add generic font-family, it is used in case of problems with external font